### PR TITLE
Feature/revsdl 2418 support service

### PR DESCRIPTION
**Pull request by bonewell**

### CODE REVIEW

File: src/components/vr_module/include/vr_module/vr_module.h [hunk 1]
```diff
@@ -82,19 +84,52 @@ class VRModule
...
+    supported_ = value;
+  }
 
+ private:
   /**
    * @brief Subscribes plugin to mobie rpc messages
    */
   void SubscribeToRpcMessages();
 
  private:
```
Comment (giangtqh) [inline src/components/vr_module/include/vr_module/vr_module.h]: remove duplicate private:

**Comment by KKarlash** [comment]
Reviewed

**Comment by bonewell** [comment]
@KKarlash 
@DKlimenko 
@VladSemenyuk 
@giangtqh 
Little changes in commands system. Review please.

**Comment by KKarlash** [comment]
complete

**Comment by giangtqh** [comment]
Reviewed

**Comment by DKlimenko** [comment]
reviewed

**Pull request merged**